tools/owncloud-unlockadmin.sh

```diff
@@ -7,8 +7,9 @@
 # purpopses only. After this point you are on your own.
 
 source /etc/mailinabox.conf # load global vars
+source ./setup/functions.sh
 
-ADMIN=$(./mail.py user admins | head -n 1)
+ADMIN=$(./management/cli.py user admins | head -n 1)
```

Comment:
This is an interesting change. Why rerun whole management-cli for unlocking only?

Reply:
I only made the change so it would work. Running the management-cli was already there. I think what this code does is checking that the requested user is actually an Admin according to the Mail-in-a-Box administration, before unlocking the nextcloud.